pkg/e2e/build_test.go

```diff
@@ -580,3 +580,19 @@ func TestBuildSubDependencies(t *testing.T) {
 	out = res.Combined()
 	assert.Check(t, strings.Contains(out, "main  Built"))
 }
+
+func TestBuildLongOutputLine(t *testing.T) {
+	c := NewParallelCLI(t)
+
+	t.Cleanup(func() {
+		c.RunDockerComposeCmd(t, "-f", "fixtures/build-test/long-output-line/compose.yaml", "down", "--mri=local")
```

Comment:
```suggestion
		c.RunDockerComposeCmd(t, "-f", "fixtures/build-test/long-output-line/compose.yaml", "down", "--rmi=local")
```

Reply:
🥴 Fixed locally and force-pushed since GitHub buttons have bitten me in the past. Thanks for catching this!